gatsby_fp/src/components/fieldComponents/consultService.js

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+
+export default function ConsultService ({consultService}){
+  return(
+    <>
+    <div className="sectiontitle">Consulting Service</div>
```

Comment:
This wants additional indentation. Running this through a linter will allow you to fix certain formatting issues automatically.

Reply:
RFC: Now would be a good time to switch to tabs over spaces (we can set tab width in our own IDEs, making for a more accessible DX). Thoughts?

Reply:
(@mf-stevens ^)

Reply:
I agree. We might want to hold off on commenting until after the linting is committed, lest a bunch of comments be considered obsolete due to the lines having changed. Since virtually every line has at least some indentation-via-spaces that would become indentation-via-tabs.

Reply:
I'm good with that. Calling this nonblocking for now. We can set up ESLint next.